### PR TITLE
Updates for NextCloud integration

### PR DESCRIPTION
**Pull request by adavenpo**

### CODE REVIEW

File: ansible/inventory [hunk 1]
```diff
@@ -1 +1 @@
-192.168.20.183 ansible_user=pi
+192.168.1.244 ansible_user=pi
```
Comment (edwinsteele) [inline ansible/inventory]: People are going to change this based on their system - no single IP address will be correct - perhaps we should just leave it as an empty file (with a comment showing format). Thoughts?

File: ansible/roles/bootstrap/tasks/main.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 - name: Add debian signing keys, necessary for backports repo
   apt_key:
-    keyserver: keyserver.ubuntu.com
+    keyserver: hkp://keyserver.ubuntu.com:80
```
Comment (edwinsteele) [inline ansible/roles/bootstrap/tasks/main.yml]: Thanks for this. Didn't consider corporate firewalls!

File: ansible/roles/webserver-content/tasks/main.yml [hunk 3]
```diff
@@ -70,3 +82,33 @@
...
+
+- name: checkout latest owncloud lib
+  git:
+    repo: https://github.com/owncloud/pyocclient.git
+    dest: /tmp/pyocclient
+    depth: 1
+    #force=yes
+
+- name: Pull python packages
+  shell: "pip install webapp2 webob"
```
Comment (edwinsteele) [inline ansible/roles/webserver-content/tasks/main.yml]: Please use the [pip task](http://docs.ansible.com/ansible/pip_module.html). Pretty sure that that the python pip package is not installed by default (not sure what's it's called in debian), so you'll need to add that too using apt.

File: ansible/roles/webserver-content/tasks/main.yml [hunk 4]
```diff
@@ -70,3 +82,33 @@
...
+
+- name: deploy NextCloud integration static content
+  shell: "cp -a /tmp/BibleBox-nextcloud/static/* {{ biblebox_default_content_root }}"
+
+- name: deploy NextCloud integration WSGI app
+  shell: "cp -a /tmp/BibleBox-nextcloud/wsgi /var/www/wsgi"
+
+- name: Add app link to uwsgi config
+  shell: "ln -fs /var/www/wsgi/app.ini /etc/uwsgi/apps-enabled/app.ini"
```
Comment (edwinsteele) [inline ansible/roles/webserver-content/tasks/main.yml]: Please use the [file task](http://docs.ansible.com/ansible/file_module.html) to create the symlink.

**Comment by adavenpo** [comment]
Thanks, Edwin...I think this addresses your comments.  FWIW, I think it's worth leaving at least an example line in the inventory for those unfamiliar with ansible.

**Comment by edwinsteele** [comment]
Thanks Andrew. Much appreciated.

**Pull request merged**